sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Release History
 
 ## 12.7.0-preview.2 (Unreleased)
-- Fixed bug where BobContainerClient.SetAccessPolicy() would throw an exception if signed identifier permissions were not in the correct order.
+- Fixed bug where BlobContainerClient.SetAccessPolicy() would throw an exception if signed identifier permissions were not in the correct order.
 - Added seekability to BaseBlobClient.OpenRead().
 - Added additional info to exception messages.
+- Added CanGenerateSasUri property, GenerateSasUri() to BlobBaseClient, BlobClient, BlockBlobClient, AppendBlobClient, PageBlobClient and BlobContainerClient.
+- Added CanGenerateSasUri property, GenerateSasUri() to BlobServiceClient.
+- Deprecated proeprty BlobSasBuilder.Version and when generating SAS will always use the latest storage SAS version of the Storage service version supported.
```

Comment:
*property

Reply:
I think this sentence could be reworked a little. Theres a lot of redundancy in it